
An unexpected error occurred:
Traceback (most recent call last):
  File "C:\Users\Dizzy\Desktop\NexG3n FireWall Manager\N3xG3n_FireWall_Manager.py", line 532, in main
    app = FirewallManagerApp(root)
  File "C:\Users\Dizzy\Desktop\NexG3n FireWall Manager\N3xG3n_FireWall_Manager.py", line 288, in __init__
    self.start_session_monitor()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'FirewallManagerApp' object has no attribute 'start_session_monitor'

